server/controllers/tournament/createTournament.ts

```diff
@@ -0,0 +1,13 @@
+import { catchAsync } from '../../utils/catchAsync';
+
+const createTournament = catchAsync(
+  (req, res, next) => {
+    req;
+    res;
+    next;
+
+    // waiting for database models
+  }
+);
+
+export default createTournament;
```

Comment:
Why the body of this function was delted? As I said earlier, I'd like to leave this function as It was. It was so general that It doesn't require any knowledge about database models.

Reply:
Hmmm this is new file, so there was not a function

Reply:
My mistake. Thought earlier somehow that it is catchAsync declaration.